stable/unifi/templates/stun-svc.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "unifi.fullname" . }}-stun
+  labels:
+    app: {{ template "unifi.name" . }}
+    chart: {{ template "unifi.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  type: {{ .Values.stunService.type }}
+  ports:
+    - port: {{ .Values.stunService.port }}
+      targetPort: stun
+      protocol: UDP
+      name: stun
```

Comment:
Same as for the controller and discovery - if a specific nodePort is defined in values.yml for, nothing would pick that up as there is no check for it here.

Reply:
fixed!